test/services/return-logs/setup/submit-start-reading.service.test.js

```diff
@@ -0,0 +1,122 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, beforeEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Test helpers
+const SessionHelper = require('../../../support/helpers/session.helper.js')
+
+// Thing under test
+const SubmitStartReadingService = require('../../../../app/services/return-logs/setup/submit-start-reading.service.js')
+
+describe('Return Logs Setup - Submit Start Reading service', () => {
+  let payload
+  let session
+  let sessionData
+  let yarStub
+
+  beforeEach(async () => {
+    sessionData = {
+      data: {
+        returnReference: '12345',
+        lines: [
+          {
+            endDate: '2019-04-30T00:00:00.000Z',
+            startDate: '2019-04-01T00:00:00.000Z'
+          },
+          {
+            endDate: '2019-05-31T00:00:00.000Z',
+            startDate: '2019-05-01T00:00:00.000Z'
+          }
+        ]
+      }
+    }
+
+    session = await SessionHelper.add(sessionData)
+
+    yarStub = { flash: Sinon.stub() }
+  })
+
+  describe('when called', () => {
+    describe('with a valid payload', () => {
+      beforeEach(async () => {
```

Comment:
Don't need that async in there
```suggestion
      beforeEach(() => {
```